src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -360,7 +405,7 @@ private SwiftCompileStep makeModulewrapStep(SourcePathResolverAdapter resolver)
 
     ProjectFilesystem projectFilesystem = getProjectFilesystem();
     return new SwiftCompileStep(
-        projectFilesystem.getRootPath(), argsFilePath, ImmutableMap.of(), compilerCommand.build());
+        projectFilesystem.getRootPath(), argsFilePath, ImmutableMap.of(), compilerCommand.build(), Optional.empty(), vfsoverlayFilePath);
```

Comment:
Is passing `vfsoverlayFilePath` still necessary when `vfsoverlay` option is turned off?

Reply:
In that case, `vfsoverlayFilePath` is not used. I can change to `Optional.empty()` to avoid confusion.